base/set.jl

```diff
@@ -428,6 +428,8 @@ end
 
 Return `true` if all values from `itr` are distinct when compared with [`isequal`](@ref).
 
+`allunique` is much more efficient when the input is sorted
```

Comment:
```suggestion
`allunique` is much more efficient when the input is sorted.
```

Reply:
without any comment for the PR or functionality itself, I must admit I don't think this statement belongs in the docstring.

Reply:
Why?  If it's not documented, then the optimization can't be relied on and people will be forced to re-implement it if they want to be sure.

Reply:
I just feel like "much" is kind of hard to quantify, bad to "promise" vague things. maybe better to retreat & reword to something more pedantic like
> `allunique` may use a specialized implementation when the input is sorted

Reply:
`unique!` already uses the "much more efficient" language, but I like the suggested change